src/qibo/core/terms.py

```diff
@@ -107,65 +111,72 @@ class SymbolicTerm(HamiltonianTerm):
             from qibo.core.terms import SymbolicTerm
             sham = X(0) * X(1) + 2 * Y(0) * Y(1)
             termsdict = sham.as_coefficients_dict()
-            sterms = [SymbolicTerm(c, f) for f, c in termsdict.items()]
+            sterms = [SymbolicTerm.from_factors(c, f) for f, c in termsdict.items()]
+
+    Args:
+        coefficient (complex): Complex number coefficient of the underlying
+            term in the Hamiltonian.
+        factors (list): List of :class:`qibo.symbols.Symbol` that represent
+            the term factors.
+        matrix_map (dict): Dictionary that maps target qubit ids to a list of
+            matrices that act on each qubit.
+    """
+    """Helper constructor using the ``sympy`` expression directly.
```

Comment:
I think there is something wrong with this comment, no?

Reply:
Yes, thanks. I was going to update the docstring after removing the `.from_factors` constructor for simplification but I forgot to complete the update. It should be ok now.